modules/bootstrapped/src/generated/smithy4s/example/traits/IndirectRecursiveViaMembersTrait.scala

```diff
@@ -0,0 +1,26 @@
+package smithy4s.example.traits
+
+import smithy4s.Hints
+import smithy4s.Schema
+import smithy4s.ShapeId
+import smithy4s.ShapeTag
+import smithy4s.schema.Schema.recursive
+import smithy4s.schema.Schema.struct
+
+/** A trait that has indirect recursive references to it via members. */
+final case class IndirectRecursiveViaMembersTrait(member: Option[RecursiveMember] = None)
+
+object IndirectRecursiveViaMembersTrait extends ShapeTag.Companion[IndirectRecursiveViaMembersTrait] {
```

Comment:
I'm 99% sure about this one.

The recursive reference to this one only happens via members, and the `schema` of this type is already lazy.

So, if you initialize `IndirectRecursiveViaMembersTrait` first, everything should initialize by the time the recursive reference is used.

What happens if you try to instantiate `RecursiveMember` first?

1. run `RecursiveMember.<clinit>`
2. instantiate `hints` of `RecursiveMember`, which includes a static binding to `IndirectRecursiveViaMembersTrait`
3. run `IndirectRecursiveViaMembersTrait.<clinit>`
4. run hints here, not a big deal
5. run `schema` here. This is lazy so we're good

at least that's what I'm thinking at the moment.